app/src/main/res/values/strings.xml

```diff
@@ -255,5 +255,9 @@
     <string name="tell_me_more">Tell me more</string>
     <string name="open_youtube_external_tabs_description">Enable to open Youtube links on external app</string>
     <string name="pref_title__open_youtube_external">Youtube links</string>
+    <string name="pref_desc__sub_theme">Change the theme of your channel</string>
+    <string name="pref_title__sub_theme">Hubzilla theme settings</string>
+    <string name="pref_title_swipe_refresh">Pull to refresh</string>
+    <string name="swipe_refresh_description">Pulling down on top of page to refresh.\nYou need to restart the app for changes to take effect.</string>
 
```

Comment:
Please now use content in name, this was much differnt in last years, but now please use e.g. `change_the_name_of_your_account`